### DISCUSSION TITLE
How to cancel a cooldown

### DISCUSSION THREAD

**Discussion by ghost** [Rapptz/discord.py#9174] [Q&A]
I have the following command:

    #Slots
    @commands.hybrid_command(name='slots', description='gamble your money with slots!', aliases=["SLOTS"])
    @commands.cooldown(1, 120, commands.BucketType.user)
    async def slots(self, ctx: commands.Context, amount: int):
        await openAccount(ctx.author)
        embed = discord.Embed(title = '', color = discord.Color.red())

        balance = await updateBank(ctx.author)
        amount = int(amount)
        if amount > balance[0]:
            embed.add_field(name = 'Slots failed', value = "You don't have sufficient funds.")
            return await ctx.send(embed = embed)

        if amount < 50:
            embed.add_field(name = 'Slots failed', value = "The amount must be 50 or more.")
            return await ctx.send(embed = embed)
        
        final = []
        for i in range(3):
            a = random.choice([":heart:", ":joy:", ":moneybag:"])
            final.append(a)
            
        if final[0] == final[1] and final[1] == final[2]:
            await updateBank(ctx.author, amount)
            embed.add_field(name = 'Winner!', value = f"{final[0]}{final[1]}{final[2]} \nYou won! {amount} has been added to your wallet.")
        else:
            await updateBank(ctx.author, -1*amount)
            embed.add_field(name = 'Loser!', value = f"{final[0]}{final[1]}{final[2]} \nYou lost {amount} coins :sob:. Good luck next time!")
        await ctx.send(embed = embed)

    @slots.error
    async def slots_error(self, ctx: commands.Context, error):
        if isinstance(error, commands.MissingRequiredArgument):
            embed = discord.Embed(title = '', color = discord.Color.red())
            embed.add_field(name = 'Slots failed', value = "Please enter the amount.")
            await ctx.send(embed = embed)
            
            
How do I make it so that the cooldown essentially is canceled when any of the errors happen (both ones that are in the command and the one in @slots.error)?

**Comment by LeoCx1000** [comment]
You can use [`Command.reset_cooldown`](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Command.reset_cooldown).
